kymatio/scattering3d/frontend/torch_frontend.py

```diff
@@ -61,8 +110,7 @@ def scattering(self, input_array):
 
         input_array = input_array.reshape((-1,) + signal_shape)
 
-        x = input_array.new(input_array.shape + (2,)).fill_(0)
-        x[..., 0] = input_array
+        x = input_array
```

Comment:
Just switch to `x` or `input_array` throughout. No need to have two variables referring to the same array.